src/lib/ColumnPreferences.js

```diff
@@ -43,6 +43,20 @@ export function getPreferences(appId, className) {
   }
 }
 
+export function getColumnSort(sortBy, appId, className) {
+  let objectName = className + ':' + 'columnSort';
```

Comment:
I think `columnSort` as a `const` placed at the top would be nice. Consistent with our existing style: https://github.com/ParsePlatform/parse-dashboard/blob/master/src/lib/ColumnPreferences.js#L79